app/index.html

```diff
@@ -16,10 +16,10 @@
 </head>
 <body>
 <div class="container">
-    <img class="logo" alt="Mayden Logo" src="images/branding-icons/maydenacademy_logotype_05.18_web.svg">
+    <img class="logo" alt="iO Academy Logo" src="./images/logos/SVGs/iO_Logo_Alt.svg">
     <div class="col-sm-12 text-center">
         <div class="col-sm-offset-34col-sm-9 text-center">
-            <h3 class="index-header">Welcome to the Mayden<sup>®</sup> Academy Aptitude Test</h3>
+            <h3 class="index-header">Welcome to the iO Academy<sup>®</sup> Aptitude Test</h3>
```

Comment:
have you asked product owner about where that symbol is supposed to go and whether it's needed?

Reply:
removed as per PO guidance